src/components/toast-viewport/toast-viewport.tsx

```diff
@@ -0,0 +1,11 @@
+import classNames from 'classnames';
+import styles from './toast-viewport.module.scss';
+import * as RadixToast from '@radix-ui/react-toast';
+
+export interface ToastViewportProps {
+    className?: string;
+}
+
+export const ToastViewport = ({ className }: ToastViewportProps) => {
+    return <RadixToast.Viewport className={classNames(styles.root, className)} />;
```

Comment:
What is ToastViewport?

Reply:
A required radix primitive to show toasts.
> The fixed area where toasts appear.